script/init_py.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+this_dir=$(dirname $0)
+
+if [ -n "$GITHUB_RUN_ID" -a -d "/home/runner/.local/lib/python3.6/site-packages/pytest" ]; then
```

Comment:
how does get there the first time?

Also - we should probably use python 3.7 or 3.8. I had some weirdness with 3.6

Reply:
It gets there via the github actions "action/cache" step.

The build is on 3.6 but pyodide is on 3.7 (.4 I think). All that goes away w/ bazelification.